api/client/cluster/client.go

```diff
@@ -146,3 +147,44 @@ func (c *clusterClient) GetGossipState() *cluster.ClusterState {
 	}
 	return status
 }
+
+func (c *clusterClient) EnumerateAlerts(resource api.ResourceType) (*api.Alerts, error) {
```

Comment:
Sure I can do that. I added it because if any driver implements only the volume interface and not the clustered osd interface, they should still be able to get alerts for only volumes.